bitnami/postgresql-ha/README.md

```diff
@@ -140,6 +140,8 @@ The following table lists the configurable parameters of the PostgreSQL HA chart
 | `pgpool.pdb.create`                            | If true, create a pod disruption budget for Pgpool pods.                                                                                                             | `false`                                                      |
 | `pgpool.pdb.minAvailable`                      | Minimum number / percentage of pods that should remain scheduled                                                                                                     | `1`                                                          |
 | `pgpool.pdb.maxUnavailable`                    | Maximum number / percentage of pods that may be made unavailable                                                                                                     | `nil`                                                        |
+| `pgpool.strategy`                              | Strategy used to replace old Pods by new ones                                                                                                                        | `{}`                                                         |
```

Comment:
In order to follow a stardardized approach in all Charts, can we change that to `updateStrategy`?

Reply:
Sure, I can change it.